prime-router/metadata/hl7_mapping/ORU_R01/base/coded-element.yml

```diff
@@ -0,0 +1,45 @@
+name: ORU-R01-observation-request
+elements:
+  - name: identifier
+    value: [ '%resource.coding.code' ]
+    hl7Spec: [ '%{ceFieldPath}-1' ]
+
+  - name: text
+    value: [ '%resource.text' ]
+    hl7Spec: [ '%{ceFieldPath}-2' ]
+
+  - name: system-url
+    resource: '%resource.coding.system'
+    # Important to only check http since there are urls defined in the Linux4Health
+    # coding system map file that are http and others that are https
+    condition: '%resource is URI and %resource.value.startsWith("http")'
```

Comment:
I don't understand how this check is important.

Reply:
There are two different options where we could be pulling the system from FHIR for what could go into HL7 CE-3. One of those is a URL that can be used as a key to pull from the mapping. I checked that file in the linux4health library and some of them are stored using `http://` and others are stored using `https://` therefore to make sure what we are using here is a url, we check only `http` to catch both cases.